pkg/kubelet/nodestatus/setters.go

```diff
@@ -213,6 +213,67 @@ func MemoryPressureCondition(nowFunc func() time.Time, // typically Kubelet.cloc
 	}
 }
 
+// PIDPressureCondition returns a Setter that updates the v1.NodePIDPressure condition on the node.
+func PIDPressureCondition(nowFunc func() time.Time, // typically Kubelet.clock.Now
```

Comment:
Pass a clock.Clock in instead?  Prefer interface arguments to explicit functions. (although if you disagree, i'm not as sold on this one).